cdap-ui/app/directives/log-viewer/log-viewer.less

```diff
@@ -180,17 +177,23 @@ my-log-viewer {
           display: inline-block;
           font-weight: normal;
         }
+        .log-stats-total {
+          margin-bottom: 4px;
+          margin-left: 9px;
+        }
         .log-stats {
           font-size: @headerbar-font-size;
+          margin-right: 15px;
         }
         .stat-name {
           color: @headerbar-stat-name-color;
           margin-left: 10px;
           margin-right: 4px;
         }
         .stat-count {
-          min-width: 30px;
-          border-radius: 8px;
+          min-width: 25px;
```

Comment:
Do we require this for the counts? It automatically should be based on the number inside it right?